Makefile

```diff
@@ -28,7 +33,7 @@ release:
 .PHONY: docker
 docker:
 	@echo $(dev_build_version) > VERSION
-	docker build -t fullstorydev/grpcui:$(dev_build_version) .
+	docker build -t fullstorydev/grpcui:$(dev_build_version) . --ca
```

Comment:
Is this a mistake? If not, what does this do? I don't see anything about a `--ca` flag when I run `docker build --help`.